app/models/concerns/hyrax/solr_document_behavior.rb

```diff
@@ -117,6 +117,12 @@ def admin_set?
       hydra_model == ::AdminSet
     end
 
+    ##
+    # @return [Boolean]
+    def work?
+      Hyrax.config.curation_concerns.include? hydra_model
```

Comment:
I've taken this from FilterByType: https://github.com/samvera/hyrax/blob/main/app/search_builders/hyrax/filter_by_type.rb#L42-L52
I'm not sure if we need to worry about documents in solr with `generic_type_sim: Work` which aren't registered curation_concerns.

Reply:
what are the tradeoffs with using the `generic_type` field instead? registered concerns is a bit awkward and used in inconsistent ways in other parts of the app, so i've been avoiding using it in new code. this looks like a pretty straightforward use, so i'm 👍🏻 on it, but wondering whether `generic_type` isn't better all-around.

Reply:
This is the document returned from solr so it doesn't include the `generic_type_si[m]` field because it is only indexed.  Maybe it is worth switching this to `ssi`?